paasta_tools/contrib/kill_orphaned_docker_containers.py

```diff
@@ -12,42 +13,35 @@ def parse_args():
             ' and optionally kills them.'
         )
     )
-    parser.add_argument('-f', '--force', help="Actually kill the containers. (defaults to dry-run)")
+    parser.add_argument('-f', '--force', action="store_true",
+                        help="Actually kill the containers. (defaults to dry-run)")
     args = parser.parse_args()
     return args
 
 
-def get_running_task_ids_from_mesos_slave():
-    state = mesos_tools.get_local_slave_state()
-    frameworks = state.get('frameworks')
-    executors = [ex for fw in frameworks for ex in fw.get('executors', [])
-                 if u'TASK_RUNNING' in [t[u'state'] for t in ex.get('tasks', [])]]
-    return set([e["id"] for e in executors])
-
-
-def get_running_mesos_docker_containers(client):
-    running_containers = client.containers()
-    return [container for container in running_containers if "mesos-" in container["Names"][0]]
-
-
 def main():
     args = parse_args()
     docker_client = get_docker_client()
-    running_mesos_task_ids = get_running_task_ids_from_mesos_slave()
-    running_mesos_docker_containers = get_running_mesos_docker_containers(docker_client)
-    print running_mesos_task_ids
+    running_mesos_task_ids = mesos_tools.get_running_task_ids_from_mesos_slave()
+    running_mesos_docker_containers = get_running_mesos_docker_containers()
```

Comment:
Can `get_running_mesos_docker_containers` share the docker client that's made here?